crates/swc/tests/tsc-references/for-of55.1.normal.js

```diff
@@ -2,6 +2,6 @@
 let v = [
     1
 ];
-for (let v1 of v1){
-    v1;
+for (let v of v){
```

Comment:
This is wrong

Reply:
I'm not sure. I look into the input file https://github.com/swc-project/swc/blob/main/crates/swc_ecma_parser/tests/tsc/for-of55.ts.

Is the original input wrong already?

Reply:
I see. My bad. Thank you!